test/cypress/integration/widgets/deployments_view_spec.ts

```diff
@@ -123,6 +130,34 @@ describe('Deployments View widget', () => {
             });
         });
 
+        it('should allow changing displayed labels in columns', () => {
+            useDeploymentsViewWidget({
+                configurationOverrides: {
+                    fieldsToShow: without(widgetConfiguration.fieldsToShow, 'blueprintName', 'location')
+                }
+            });
+
+            getDeploymentsViewTable().within(() => {
+                cy.contains(deploymentName);
+                cy.contains(labelName).should('not.exist');
+            });
+
+            cy.log('Show a label');
+            cy.editWidgetConfiguration(widgetId, () => {
+                widgetConfigurationHelpers.toggleLabelsDropdown();
+                widgetConfigurationHelpers.getLabelsDropdown().within(() => {
+                    cy.get('[role="listbox"]').contains(labelName).click();
```

Comment:
Optional:

I'm not 100% sure, but probably you can use here shorter version:
```suggestion
                    cy.contains('[role="listbox"]', labelName).click();
```

See https://docs.cypress.io/api/commands/contains#Selector for details.